### PR TITLE
fix: css for quiz and leaderboard

### PR DESCRIPTION
**Pull request by KaustubhKumar05**

### CODE REVIEW

File: packages/roomkit-react/src/Prebuilt/components/Polls/Voting/Voting.tsx [hunk 1]
```diff
@@ -88,12 +88,28 @@ export const Voting = ({ id, toggleVoting }: { id: string; toggleVoting: () => v
         )}
 
         {canViewLeaderboard ? (
-          <Button
-            css={{ fontWeight: '$semiBold', w: 'max-content', ml: 'auto', mt: '$8' }}
-            onClick={() => setPollView(POLL_VIEWS.RESULTS)}
+          <Flex
+            css={{
+              position: 'absolute',
```
Comment (raviteja83) [inline packages/roomkit-react/src/Prebuilt/components/Polls/Voting/Voting.tsx:93]: why absolute?

Comment (KaustubhKumar05) [reply packages/roomkit-react/src/Prebuilt/components/Polls/Voting/Voting.tsx:93]: Seemed appropriate for the new design: https://www.figma.com/file/ELM8oi6Abp9w1dzF5ApiJA/Prebuilt-Kit---Stage-1.5?node-id=645%3A97608&mode=dev

image

Comment (raviteja83) [reply packages/roomkit-react/src/Prebuilt/components/Polls/Voting/Voting.tsx:93]: we can scroll the center part by keeping the footer in place with flex only no?

Comment (KaustubhKumar05) [reply packages/roomkit-react/src/Prebuilt/components/Polls/Voting/Voting.tsx:93]: Updated in https://github.com/100mslive/web-sdks/pull/2470

**Review by eswarclynn — APPROVED** [review]

**Pull request merged**